components/service/glean/src/main/java/mozilla/components/service/glean/storages/CountersStorageEngine.kt

```diff
@@ -43,8 +43,8 @@ internal open class CountersStorageEngineImplementation(
         amount: Int
     ) {
         // Use a custom combiner to add the amount to the existing counters rather than overwriting
-        super.recordScalar(metricData, amount) { currentValue, amount ->
-            currentValue?.let { it + amount } ?: amount
+        super.recordScalar(metricData, amount) { currentValue, newAmount ->
+            currentValue?.let { it + newAmount } ?: newAmount
```

Comment:
Is it possible this is in this PR accidentally?

Reply:
This fixes the variable shadowing as mentioned in the initial commit message. Should I have a separate PR for this?

Reply:
Oh, I just missed that in the PR comment.  Not a big deal to include here.